charts/alfresco-search-service/values.yaml

```diff
@@ -132,15 +127,12 @@ initContainer:
       memory: "5Mi"
     limits:
       memory: "10Mi"
-
 global:
   alfrescoRegistryPullSecrets: quay-registry-secret
-
 nodeSelector: {}
 tolerations: []
 # -- Pod affinity, passed thru tpl function
-affinity: |-
-
+affinity: ""
```

Comment:
raised https://github.com/updatecli/updatecli/issues/1875

Reply:
I'm fine with that